src/posts/PublishedDate.js

```diff
@@ -3,7 +3,10 @@ import React from 'react';
 class PublishedDate extends React.Component{
     render(){
         return(
-            <p>{this.props.publishedDate.toString()}</p>
+            <div className="left-align-board">
```

Comment:
`className="left-align-board"` by itself won't left align the board. You will need to use CSS, or Material-UI's [styling feature](https://material-ui.com/styles/basics/). Give this a read.